client/my-sites/site-settings/jetpack-module-toggle.jsx

```diff
@@ -45,18 +45,19 @@ class JetpackModuleToggle extends Component {
 
 	handleChange = () => {
 		if ( ! this.props.checked ) {
-			this.recordTracksEvent( 'calypso_jetpack_module_toggle_activated' );
```

Comment:
I'm not sure what the story is around updating these events. `_activated` seems to be used in a few funnels at least.

I think these need to be marked as deprecated now: 1185-gh-tracks-events-validation-status

Do you know whether we can get existing data rewritten to match `calypso_jetpack_module_toggle` format?

Reply:
> Do you know whether we can get existing data rewritten

I do not think it is necessary:
- there are only 3 funnels with that event https://mc.a8c.com/tracks/events/calypso_jetpack_module_toggle_activated/
- since we merged significant changes to the module handling in wp-admin (manual activation), analyses that include previous data would be confounded— it is best to use this data post-modularization.

Reply:
just circling back on this: considering rewriting historical data is a pain, unless we see real value in it ( like we are thinking about actually analysing historical data ) I'd discourage it :)

Reply:
To the best of my knowledge and understanding, considering the changes which mark coherent analysis, there is no value in keeping this historical data. Pinging @johnHackworth as an fyi, as the funnels that contained the previous version were computed by him.